crates/server/src/admin/service.rs

```diff
@@ -39,6 +39,8 @@ use crate::admin::handlers::root_keys::{create_root_key_handler, delete_auth_key
 use crate::config::ServerConfig;
 use crate::middleware;
 use crate::middleware::auth::AuthSignatureLayer;
+#[cfg(feature = "host_layer")]
```

Comment:
This should be separate PR

Reply:
https://github.com/calimero-network/core/pull/712